src/components/Navbar.js

```diff
@@ -0,0 +1,60 @@
+import {connect} from "react-redux";
+import React, {Component} from "react";
+import "../style/pages/logIn.css"
+import {isAuth, withNavigation} from "../util/heleper";
+import {LOG_IN, USER_EMAIL} from "../constants";
+
+
+class Navbar extends Component {
+
+    render() {
+
+        const {email} = this.props;
+
+        return withNavigation(window.location.pathname) ? (
+            <nav className="navbar navbar-expand-lg navbar-light bg-light">
+                <a className="navbar-brand" href="#">
+                    <s1>LOGO</s1>
+                    <s2>TYPE</s2>
+                </a>
+                <button className="navbar-toggler" type="button" data-toggle="collapse"
+                        data-target="#navbarSupportedContent"
+                        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span className="navbar-toggler-icon"></span>
+                </button>
+
+                <div className="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul className="navbar-nav mr-auto">
+
+                    </ul>
+                    {isAuth() ? (
```

Comment:
https://medium.com/@tomlarge/private-routes-with-react-router-dom-28e9f40c7146
Read something like this please.